tests/executors/test_celery_executor.py

```diff
@@ -363,8 +360,8 @@ def test_check_for_stalled_adopted_tasks(self):
             task_1 = BaseOperator(task_id="task_1", start_date=start_date)
             task_2 = BaseOperator(task_id="task_2", start_date=start_date)
 
-        key_1 = TaskInstanceKey(dag.dag_id, task_1.task_id, exec_date, try_number)
-        key_2 = TaskInstanceKey(dag.dag_id, task_2.task_id, exec_date, try_number)
+        key_1 = TaskInstanceKey(dag.dag_id, task_1.task_id, "runid", try_number)
+        key_2 = TaskInstanceKey(dag.dag_id, task_2.task_id, "runid", try_number)
```

Comment:
Less sure about this; is creating real DagRun and TaskInstance objects worthwhile?

Reply:
Might be worth it, yes. Two reasons I did this: it was late, and nothing in the Executor cares about DagRun or strictly speaking executor_date.


Thoughts?